src/components/action-bar/action-bar-item/action-bar-overflow-menu.tsx

```
@@ -44,17 +44,19 @@ export class ActionBarOverflowMenu {
-                    {this.getOverflowTriggerContent()}
-                </button>
-            </limel-menu>,
-        ];
+        return (
+            <Host>
+                <limel-menu
+                    openDirection={this.openDirection}
+                    items={this.items}
+                    onSelect={this.handleSelect}
+                >
+                    <button slot="trigger">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)